### PR TITLE
(MODULES-3406) Optimize PowerShell parse time

### PR DESCRIPTION
**Pull request by Iristyle**
Make some simple code refactors to execute faster - in the unit tests this improved execution time by roughly 15%

### CODE REVIEW

File: lib/puppet_x/templates/init_ps.ps1.erb [hunk 1]
```diff
@@ -286,8 +286,8 @@ function New-XmlResult
   $xml = [xml]@"
 <ReturnResult>
   <Property Name='exitcode'>$($exitcode)</Property>
-  <Property Name='errormessage'>$([System.Convert]::ToBase64String([System.Text.Encoding]::UTF8.GetBytes([string]$errormessage)))</Property>
```
Comment (jpogran) [inline lib/puppet_x/templates/init_ps.ps1.erb]: Already went down this road. You're going to want to keep the fully qualified namespace to support PowerShell v2.

**Comment by ferventcoder** [comment]
Blocked until @glennsarti has merged master to stable, then we can pull this in.

**Comment by jpogran** [comment]
Commenting on non hidden commit:

Already went down this road. You're going to want to keep the fully qualified namespace to support PowerShell v2.

File: lib/puppet_x/templates/init_ps.ps1.erb (hunk 1 above)
Comment (Iristyle) [inline lib/puppet_x/templates/init_ps.ps1.erb]: Thanks for the heads up... that is weird!

**Comment by Iristyle** [comment]
Closing in favor of PR #115, which was targeted at master

**Pull request closed without merging**